core/unit_test/CMakeLists.txt

```diff
@@ -23,7 +26,7 @@ ENDIF()
 
 TARGET_INCLUDE_DIRECTORIES(kokkos_gtest PUBLIC ${GTEST_SOURCE_DIR})
 #Gtest minimally requires C++11
-TARGET_COMPILE_FEATURES(kokkos_gtest PUBLIC cxx_std_11)
+#TARGET_COMPILE_FEATURES(kokkos_gtest PUBLIC cxx_std_11)
```

Comment:
What's the deal with this?

Reply:
Needs fixing. But that didn't pass.

Reply:
why did we need this in the first place again? Didn't seem to trip up any of our tests?

Reply:
For compilers for which c++11 is not a default. The gtest code needs C++11. Are we no longer testing 4.8.5 or anything else that doesn't have 11 by default?

Reply:
that is tested and it passed?

Reply:
Hmm. Looks like we do have GCC 4.8.4 in the tests. I don't know how it passed. This was missing before and builds were failing because `-std=c++11` was missing for `gtest` was missing.